doc/source/user_guide/cmdlang_native.rst

```diff
@@ -94,26 +94,41 @@ as parameters, which means that this syntax is also valid:
     session.ensight.part.modify_end()
 
 
+Native API debugging
+^^^^^^^^^^^^^^^^^^^^
+
 Every command also returns an error code, which is ``0`` on success. For example,
 :samp:`err = session.ensight.part.colorby_rgb([0.0,0.0,"sad"])` sets ``err`` to ``-1``.
-This code shows how you can also arrange to have error return values converted
+The :func:`attrtree<ansys.api.pyensight.ensight_api.ensight.sendmesgoptions>`
+method can be using to use Python exception handling instead of

```

Comment:
@randallfrank "can be using to use"? I think you meant "can be set to use".

Reply:
LOL, done!